library/src/main/java/pl/netigen/rewards/RewardListenersList.java

```diff
@@ -0,0 +1,22 @@
+package pl.netigen.rewards;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RewardListenersList extends ArrayList<RewardsListener> {
```

Comment:
Adding listener should check if listener is not already added to list

Reply:
Done, check is added in overriden add method.